tests/grouping/test_grouping_utils.py

```diff
@@ -294,3 +299,154 @@ def test_qwc_complement_adj_matrix_exception(self):
 
         with pytest.raises(ValueError, match="Expected a binary array, instead got"):
             qwc_complement_adj_matrix(not_binary_observables)
+
+    @pytest.mark.parametrize(
+        "pauli_word,wire_map,expected_string",
+        [
+            (PauliX(0), {0: 0}, "X"),
```

Comment:
Worth having an example where the values in the `wire_map` are non-consecutive integers.

This applies to other test cases too.

Reply:
Have added cases to each that include non-consecutive integer wire maps, as well as Paulis with their tensor products expressed in an order different than the map order. Let me know if you spot anything unusual or if you notice I missed a function.